srcpkgs/dtools/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'dtools'
+pkgname=dtools
+# keep this synchronized with libphobos and dmd
+version=2.081.1
+revision=1
+build_wrksrc="tools-${version}"
+create_wrksrc=yes
```

Comment:
you likely simply need to specify `wrksrc="tools-v${version}"` rather than `build_wrksrc` and `create_wrksrc`.

Reply:
alright